docs/clients/promtail/stages/regex.md

```diff
@@ -87,4 +87,3 @@ While the regex stage would then parse the value for `time` in the extracted map
 and append the following key-value pairs back into the extracted map:
 
 - `year`: `2019`
-
```

Comment:
?

Reply:
There are 2 new lines after `year: 2019`, removing the last line shows this extra character. Kind of weird. Anyways, I reverted this change.